metaflow/client/core.py

```diff
@@ -570,6 +622,48 @@ def traverse(cls, ids_r, lst):
             self._path_components = traverse(_CLASSES[self._NAME], ids, [])
         return self._path_components
 
+    def add_tag(self, tags):
+        """
+        Add one or more tags to the current object
+
+        Parameters
+        ----------
+        tags : string or List[string]
```

Comment:
`List[string]` is not correct - any iterable is accepted (which is desirable) except strings are handled specially